src/back/space/src/api/list-available.cpp

```diff
@@ -24,17 +26,19 @@ formats::json::Value ListAvailable::HandleRequestJsonThrow(
 	formats::json::ValueBuilder res;
 
 	try {
-		const auto userId = req.GetHeader(headers::kUserId);
-		if (userId.empty())
-			throw errors::Unauthorized401{};
+		const auto params = ValidateRequest(_mapHttpMethodToSchema, req, body);
+		const auto userId = params[headers::kUserId].As<std::string>();
 
-		const auto paging = parsePaging(req);
+		Paging paging = {
+			.start = params["start"].As<int>(),
+			.limit = params["limit"].As<int>()
+		};
```

Comment:
Посмотри как в МС Проекты парсятся параметры пагинации
```suggestion
		const auto paging = parsePaging(params);
```

Reply:
Готово (везде в этом мс).